store/datastore.go

```diff
@@ -32,7 +32,7 @@ type Storer interface {
 	UpdateDataset(ID string, dataset *models.Dataset) error
 	UpdateDatasetWithAssociation(ID, state string, version *models.Version) error
 	UpdateDimensionNodeID(dimension *models.DimensionOption) error
-	UpdateEdition(datasetID, edition, state string) error
+	UpdateEdition(datasetID, edition string, version *models.Version) error
```

Comment:
Consider calling the version parameter `latestVersion` to give it some context of its purpose.